opentreemap/treemap/views/tree.py

```diff
@@ -90,16 +90,59 @@ def search_tree_benefits(request, instance):
     }
 
     formatted = format_benefits(instance, benefits, basis, digits=0)
-    formatted['hide_summary'] = hide_summary
 
-    formatted['tree_count_label'] = (
-        'tree,' if basis['plot']['n_total'] == 1 else 'trees,')
-    formatted['plot_count_label'] = (
-        'planting site' if basis['plot']['n_plots'] == 1 else 'planting sites')
-    if instance.has_resources and 'resource' in basis:
-        formatted['plot_count_label'] += ','
+    n_trees = basis['plot']['n_total']
+    n_plots = basis['plot']['n_plots']
+    n_empty_plots = n_plots - n_trees
+    n_resources = 0
+
+    tree_count_label = _('tree') if n_trees == 1 else _('trees')
+    tree_count_label += ','
+    empty_plot_count_label = (
+        _('empty planting site') if n_empty_plots == 1 else
+        _('empty planting sites'))
```

Comment:
Note that the Django docs recommend not comparing against 1 for pluralization, but using the support built-in to `ungettext` for pluralization (to support languages with different suffixes for 2, 3, etc.) https://docs.djangoproject.com/en/1.10/topics/i18n/translation/#pluralization

I think that would make this code looks like this:
```python
tree_count_label = ungettext('tree', 'trees', n_trees)
empty_plot_count_label = _('empty planting site', 'empty planting sites', n_empty_plots)
```